deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts

```diff
@@ -215,11 +216,13 @@ export class MigrationConsoleStack extends MigrationServiceCore {
                 "cloudwatch:GetMetricData"
             ]
         })
+        const sourceClusterEndpoint = props.sourceClusterEndpoint ?? `https://capture-proxy-es.migration.${props.stage}.local:9200`;
 
         // Upload the services.yaml file to Parameter Store
         let servicesYaml = props.servicesYaml
         servicesYaml.source_cluster = {
-            'endpoint': `https://capture-proxy-es.migration.${props.stage}.local:9200`,
+            'endpoint': sourceClusterEndpoint,
+            // TODO: We're not currently supporting auth here, this may need to be handled on the migration console
```

Comment:
My stance here has been that we need to require this information initially with the CDK (in a future change)